src/particles/text/component.tsx

```diff
@@ -64,26 +64,49 @@ export const getTextTag = (variant?: string): TextTag => {
 interface IStyledTextProps {
   theme: ITextTheme;
   alignment?: TextAlignment;
+  clipToLines?: number;
 }
 
 const StyledText = styled.span<IStyledTextProps>`
   ${(props: IStyledTextProps): string => themeToCss(props.theme)};
   ${(props: IStyledTextProps): string => (props.alignment ? `text-align: ${props.alignment}` : '')};
+
+  &.singleLine {
+    white-wrap: nowrap;
+    text-overflow: ellipses;
+    overflow: hidden;
+  }
+
+  &.fixedLines {
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    ${(props: IStyledTextProps): string => (props.clipToLines ? `-webkit-line-clamp: ${props.clipToLines};` : '')}
+    overflow: hidden;
+  }
 `;
 
 export interface ITextProps extends IComponentProps<ITextTheme>, ISingleAnyChildProps {
   alignment?: TextAlignment;
   tag?: TextTag;
+  clipToLines?: number;
 }
 
 export const Text = (props: ITextProps): React.ReactElement => {
   const theme = useBuiltTheme('texts', props.variant, props.theme);
+
+  let clipToLines = props.clipToLines;
+  if (props.clipToLines && props.clipToLines <= 0) {
```

Comment:
i dont think this would work for 0

Reply:
yeah, so I had set it to undefined so the classes don't get applied.